src/Command/EncryptionDumpFieldsMapCommand.php

```diff
@@ -76,9 +76,24 @@ protected function execute(InputInterface $input, OutputInterface $output): int
                 continue;
             }
 
+            // The min/max query options must have the same type as the field.
+            // But the PHP driver always convert to "int" or "float" when the value fit in the range
             foreach ($encryptedFieldsMap as $ns => $encryptedFields) {
+                $fields = json_decode(PackedArray::fromPHP($encryptedFields['fields'])->toCanonicalExtendedJSON(), true);
```

Comment:
Would it be prudent to specify [`JSON_THROW_ON_ERROR`](https://www.php.net/manual/en/json.constants.php#constant.json-throw-on-error) here. It looks like that is commonly done [in Symfony](https://github.com/search?q=repo%3Asymfony%2Fsymfony%20JSON_THROW_ON_ERROR&type=code).

Reply:
The JSON comes from `toCanonicalExtendedJSON`; it will be always valid.